### PR TITLE
Fix/dropdown disabled and selected options

### PR DESCRIPTION
**Pull request by mario-subo**
Multi select:

before:

Screenshot 2021-04-05 at 2 56 49 PM

After:

Screenshot 2021-04-05 at 2 57 36 PM

### CODE REVIEW

File: lib/src/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -79,14 +79,32 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(({ wrapperProp
...
+    );
+
+    const isAllSelected = (): boolean => {
+        return Array.from(selectRef.current.options).every((_, i) => {
+            const option = selectRef.current.options.item(i);
+            if (option.disabled) {
+                return true;
+            } else {
+                return option.selected;
+            }
```
Comment (omarVodiaK) [inline lib/src/Dropdown/Dropdown.tsx:106]:
```suggestion
          option.disabled ? true : option.selected
```

**Review by omarVodiaK — APPROVED** [review]

File: lib/src/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (kherP) [reply lib/src/Dropdown/Dropdown.tsx:106]: do we need that true? option.disabled || option.selected

**Pull request merged**